netcdf/src/file.rs

```diff
@@ -216,7 +215,12 @@ impl File {
     ///
     /// Not a `netCDF-4` file
     pub fn group<'f>(&'f self, name: &str) -> error::Result<Option<Group<'f>>> {
-        super::group::group_from_name(self.ncid(), name)
+        super::group::group_from_path(self.ncid(), name.split('/')).map(|ncid| {
```

Comment:
Use `get_subgroup_ncid_and_stem_from_path` here to only return `None` if the last group is missing